src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -40,13 +42,19 @@
 
     private final PloidyModel ploidyModel;
 
+    private final int MAX_GENOTYPE_COUNT_TO_ENUMERATE = configuration.genotypeArgs.MAX_GENOTYPE_COUNT;
```

Comment:
Also, this shouldn't be in all caps, since it isn't really a constant, it depends on the value of MAX_GENOTYPE_COUNT which also isn't really a constant since it's set on the commandline.